libraries/dotnet/EnergyOrigin.TokenValidation/EnergyOrigin.TokenValidation.Unit.Tests/Options/TokenValidationOptionsTests.cs

```diff
@@ -0,0 +1,57 @@
+using System.ComponentModel.DataAnnotations;
+using EnergyOrigin.TokenValidation.Options;
+
+namespace EnergyOrigin.TokenValidation.Unit.Tests.Options;
+
+public class TokenValidationOptionsTests
+{
+    [Fact]
+    public void Properties_ShouldBeInitializedAsNull()
+    {
+        var options = new TokenValidationOptions();
+
+        Assert.Null(options.PublicKey);
+        Assert.Null(options.Issuer);
+        Assert.Null(options.Audience);
+    }
+
+    [Fact]
+    public void Properties_ShouldBeCorrectlyInitialized()
+    {
+        var publicKey = new byte[] { 1, 2, 3 };
+        var issuer = "TestIssuer";
+        var audience = "TestAudience";
+
+        var options = new TokenValidationOptions
+        {
+            PublicKey = publicKey,
+            Issuer = issuer,
+            Audience = audience
+        };
+
+        Assert.Equal(publicKey, options.PublicKey);
+        Assert.Equal(issuer, options.Issuer);
+        Assert.Equal(audience, options.Audience);
+    }
+
+    [Theory]
+    [InlineData(null, "TestIssuer", "TestAudience")]
+    [InlineData(new byte[0], null, "TestAudience")]
+    [InlineData(new byte[0], "TestIssuer", null)]
+    public void Properties_ShouldFailValidation_WhenRequiredPropertyIsNull(byte[] publicKey, string issuer, string audience)
+    {
+        var options = new TokenValidationOptions
+        {
+            PublicKey = publicKey,
+            Issuer = issuer,
+            Audience = audience
+        };
+
+        var validationResults = new List<ValidationResult>();
+        var validationContext = new ValidationContext(options, null, null);
+        var isValid = Validator.TryValidateObject(options, validationContext, validationResults, true);
+
+        Assert.False(isValid);
+        Assert.NotEmpty(validationResults);
```

Comment:
You should check equals with the validation success property instead

Reply:
Like this?: https://github.com/Energinet-DataHub/energy-origin/pull/513/commits/214872e553cafd4e129295bb87a811f08d0f82da

Reply:
Well no, I mean check equals against this property: https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.validationresult.success?view=net-8.0

Reply:
Like this then?:

https://github.com/Energinet-DataHub/energy-origin/pull/513/commits/90a3893630db1ed99d4489fa2542cff54659517f